README.md

```diff
@@ -1 +1,55 @@
-# Hello_Microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+### Project Name
+
+Hello_Microverse
+
+
+## Built With
+
+-CSS
+- Github workflow
+- Linters
+- HTML
+
+## Live Demo (if available)
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+## Authors
+
+👤 **Blessman Newton**
+
+- GitHub: [@githubhandle](https://github.com/Blessman-Newton)
+- Twitter: [@twitterhandle](https://twitter.com/Blessman_Newton)
+- LinkedIn: [LinkedIn](www.linkedin.com/in/Blessman)
+
+
+## 🤝 Contributing
+
+Contributions and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a 🤝 if you like this project!
+
+## Acknowledgments
+
+- Hat tip to microverse and their curriculum github setup
+
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
Hey 👋🏽 as requested by the previous reviewers, your README is lacking several required sections of the template to make it look really professional 😊 compare your current version to the [template](https://github.com/microverseinc/readme-template) and make sure you add the missing sections ❌ you're only allowed to remove the `FAQ` because it's optional, the `Live Demo` because it doesn't apply and some sections in `Getting Started` that aren't relevant to this project yet such as `Usage`, Run tests` and `Deployment`. 🤓

Reply:
Hello, Thank for the feedback correction.
I will like you to help me go through the changes I have made.